File: BiznetDISoft_WS_2_0_0/App_Code/ParameterReturnValue.vb
Imports Microsoft.VisualBasic

Public Class ParameterReturnValue
    Private _ParaName As String
    Private _ParaValue As String
    Private _ParaIndex As Integer

    Public Sub New()
        _ParaName = ""
        _ParaValue = ""
        _ParaIndex = -1
    End Sub
    Public Property ParameterName() As String
        Get
            ParameterName = _ParaName
        End Get
        Set(ByVal value As String)
            _ParaName = value
        End Set
    End Property

    Public Property ParameterValue() As Object
        Get
            ParameterValue = _ParaValue
        End Get
        Set(ByVal value As Object)
            _ParaValue = value
        End Set

    End Property
    Public Property ParameterIndex() As Integer
        Get
            ParameterIndex = _ParaIndex
        End Get
        Set(ByVal value As Integer)
            _ParaIndex = value
        End Set
    End Property


End Class

File: DatabaseOperation/DBContract/ParameterReturnValue.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DatabaseOperation.DBContract
{
    public class ParameterReturnValue
    {
        private string _ParaName;
        private object _ParaValue;
        private int _ParaIndex;

        public ParameterReturnValue()
        {
            _ParaName = "";
            _ParaValue = "";
            _ParaIndex = -1;
        }

        public string ParameterName
        {
            get { return _ParaName; }
            set { _ParaName = value; }
        }

        public object ParameterValue
        {
            get { return _ParaValue; }
            set { _ParaValue = value; }
        }

        public int ParameterIndex
        {
            get { return _ParaIndex; }
            set { _ParaIndex = value; }
        } 


    }
}
